.travis.yml

```diff
@@ -51,6 +51,7 @@ jobs:
       cache:
         pip: true
         directories:
+          - dist
```

Comment:
This should be revisited. At this point it was already deployed, right?

Reply:
Yeah, I also forgot that the dist folder is actually not even checked with proselint at all. Just `src/content`. So this is just useless